src/styles.css

```
@@ -5,7 +5,7 @@
 
 @custom-variant dark (&:is(.dark *));
 
-@import '@/modules/app/styles/index.css';
+@import './modules/app/styles/index.css';
```

Stylelint (17.6.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)